### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [all3n/all3n.github.io#1] [General]
# about/

About Me2009开始接触编程，初期是主要做php web，2011-2013 做php，14年开始转etl 做了2年，16年至今从事算法相关工作。 1just want coding like R&amp;B style!!&quot;);  Skills Pragraming Languages  java 8.5 scala 8 python 8 pig 7 PHP 7 Javascr

http://localhost:4000/about/

**Comment by all3n** [comment]
test
